front/app/components/IdeaCards/IdeasWithFiltersSidebar/index.tsx

```diff
@@ -264,19 +241,6 @@ const IdeasWithFiltersSidebar = ({
     setFiltersModalOpened(false);
   }, []);
 
-  const [isCTABarVisible, setIsCTABarVisible] = useState(false);
-
-  useEffect(() => {
```

Comment:
This was also fired, and causing re-renders, when scrolling on mobile, while we don't need it there. That's another benefit we get of moving this to `ContentRight`; it will only fire on desktop.